website/docs/r/appautoscaling_policy.html.markdown

```diff
@@ -89,6 +89,37 @@ resource "aws_ecs_service" "ecs_service" {
 }
 ```
 
+### Aurora Read Replica Autoscaling
+
+```hcl
+resource "aws_appautoscaling_target" "replicas" {
+  service_namespace  = "rds"
+  scalable_dimension = "rds:cluster:ReadReplicaCount"
+  resource_id        = "cluster:aurora-cluster-id"
```

Comment:
Nitpick: we should point to the resource for clarity 😉  `"cluster:${aws_rds_cluster.example.id}"`